File: src/codes/Users_Manual/gain/group_0.py
~from pyautd3 import Focus, FocusOption, Null, GainGroup
~x = 1.0
~y = 0.0
~z = 0.0
GainGroup(
    key_map=lambda _: lambda tr: "null" if tr.idx() <= 100 else "focus",
    gain_map={"null": Null(), "focus": Focus(pos=[x, y, z], option=FocusOption())},
)

File: src/codes/Users_Manual/gain/group_0.cs
~using AUTD3Sharp.Utils;
~using AUTD3Sharp.Gain;
~using AUTD3Sharp.Driver.Datagram;
~var x = 0.0f;
~var y = 0.0f;
~var z = 0.0f;
new Group(
    keyMap: dev => tr => tr.Idx() <= 100 ? "null" : "focus",
    gainMap: new Dictionary<object, IGain> {
        { "null", new Null() },
        { "focus", new Focus(pos: new Point3(x, y, z), option: new FocusOption()) }
    }
);